### DISCUSSION TITLE
Line chart gradient direction

### DISCUSSION THREAD

**Discussion by ber4444** [patrykandpatrick/vico#405] [Ideas]
<img width="293" alt="Screenshot 2022-11-29 at 3 14 47 PM" src="https://user-images.githubusercontent.com/205867/204669924-8e72ac29-2f4a-4b52-bcaf-5fba73c57c8d.png">

I see in the samples how you can change the color of the line shadow, but how do I reverse its direction for the red columns and make it stop at the `ThresholdLine`?

**Comment by ber4444** [comment]
not a critical feature, feel free to reopen if you get inspired to work on it though

**Comment by ghost** [comment]
Thanks for the suggestion! We’ll look into it.

**Comment by Gowsky** [comment]
Hello, @ber4444. This is possible in Vico 2. It was introduced in `2.0.0-alpha.1` with `TopBottomShader` and made more customizable in `2.0.0-alpha.26` with `AreaFill` and its `double` factory function. It is the default behavior.
